
##### Action space #####
n_actions: 3 # thumb, index, and mrl

index_vel_ub: 1.0 # Upper bound
index_vel_lb: -1.0 # Lower bound

thumb_vel_ub: 1.0
thumb_vel_lb: -1.0

mrl_vel_ub: 1.0
mrl_vel_lb: -1.0

##### Observation space #####
delta_pos: 0.01 # Position increment in radians

# Taken from mia_hand_description - Joints Limits of hand MNS20
index_pos_ub: 1.188
index_pos_lb: -1.177

thumb_pos_ub: 1.313
thumb_pos_lb: 0.029

mrl_pos_ub: 1.303
mrl_pos_lb: -0.120


